### PR TITLE
Navigation Bar

### PR DESCRIPTION
**Pull request by kmatan99**

### CODE REVIEW

File: src/components/searchBox/style.module.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    border-radius: 12px;
+    border: white;
+    padding: 5px;
+}
+
+.container {
+    display: flex;
+    flex-direction: row;
+    margin: auto;
+    margin-right: 600px;
```
Comment (iivanu) [inline src/components/searchBox/style.module.css:13]: delete line, margin: auto is enough.

Comment (iivanu) [inline src/components/searchBox/style.module.css:13]: delete line, margin: auto is enough

Comment (iivanu) [inline src/components/searchBox/style.module.css:13]: delete this line, margin: auto is enough.

File: src/components/searchBox/style.module.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+}
```
Comment (ivanivanusic) [inline src/components/searchBox/style.module.css]: delete margin-right, center margin is enough

**Review by ivanivanusic — COMMENTED** [review]
Looks good for me, just fix commented line.

**Pull request merged**